src/coreComponents/constitutive/unitTests/testMultiFluid.cpp

```diff
@@ -698,6 +801,240 @@ TEST_F( DeadOilFluidFromTableTest, numericalDerivativesMolar )
   }
 }
 
+template class MultiPhaseMultiComponentFluid< PVTProps::BrineCO2Density,
+                                              PVTProps::BrineViscosity,
+                                              PVTProps::SpanWagnerCO2Density,
+                                              PVTProps::FenghourCO2Viscosity,
+                                              PVTProps::CO2Solubility >;
```

Comment:
I hoped this would not be needed in the test files... can you confirm?

Reply:
Thanks for the catch, I forgot to remove this, and it broke the compilation for the non-GPU builds on Travis... I removed it in the latest commit.